charts/helmet/values.yaml

```diff
@@ -402,10 +402,14 @@ exports:
     ## @param envVars Environment variables to be set on APP container
     ## Example:
     ## envVars:
-    ##   - name: FOO
+    ##   FOO: "foo"
+    ##   BAR:
     ##     value: "bar"
```

Comment:
Based on this comment https://github.com/companyinfo/helm-charts/pull/24/files#r1355745000, you can remove these lines:
```suggestion
```

Reply:
Technically, this does work (albeit as a bit of a side effect of how valuesFrom works) since we render the whole value of the `BAR` key into something like: 
```yaml
env:
- name: BAR
  value: "bar"
```

But I'm happy not to advertize it as something people should be doing.